src/App.js

```diff
@@ -31,6 +29,7 @@ import { collection, doc, getDocs } from "firebase/firestore";
 import MapComponent from "./components/map/ITEMmap";
 import { db } from "./firebase";
 import SellerForm from "./pages/SellerRegistration";
+import ScrollToTop from "./components/ScrollToTop"; // Import the ScrollToTop component
```

Comment:
Hey @kushagra-singhh, Under the components directory, create a new folder named ScrollToTop.

Reply:
Discussed with @PayalKumari10.

Hey @kushagra-singhh, 
Could you convert this component to a helper/util function ? which can be called as and when required to ?

Possible Usecases:
1. If we want to add a button to scrollToTop, we can call the function on click of that button.
2. We can directly call the scrollToTop helper/util function in useEffect of any existing component.

Do add anything if I missed @PayalKumari10.

Reply:
@akbatra567 , Your approach is correct.